contracts/fungible-tokens/andromeda-cw20-staking/src/contract.rs

```diff
@@ -457,6 +602,16 @@ fn execute_claim_rewards(ctx: ExecuteContext) -> Result<Response, ContractError>
                 };
                 msgs.push(asset.transfer_msg(sender)?);
             }
+            if !token.is_active {
+                let reward_balance = token
+                    .asset_info
+                    .query_balance(&deps.querier, &env.contract.address)?;
+                // Note: Due to small rounding error, the reward_balance and rewards can be unequal even if all the rewards are distributed
+                // TODO: Remove inactive reward token in case the remaining balance is below the low limit
```

Comment:
What's the status of this TODO?

Reply:
Just updated the logic to handle rounding error